clients/src/test/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImplTest.java

```diff
@@ -764,6 +766,44 @@ public void testLeaveGroupWhenMemberOwnsAssignment() {
         testLeaveGroupReleasesAssignmentAndResetsEpochToSendLeaveGroup(membershipManager);
     }
 
+    @Test
+    public void testLeaveGroupWhenAssignmentEmpty() {
```

Comment:
Is there maybe a more direct way to test the bug?
I had a hard time to understand this unit test.
Nevermind if there is no more direct way.

Reply:
Replaced it by a simpler test